google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -294,24 +241,14 @@ private void resetModuleConfigSelection(Project project) {
     }
   }
 
-  private void refreshApplicationInfoPanel() {
-    if (gcpProjectSelector.getProject() != null && gcpProjectSelector.getSelectedUser() != null) {
-      appInfoPanel.refresh(gcpProjectSelector.getProject().getProjectId(),
-          gcpProjectSelector.getSelectedUser().getCredential());
-    } else {
-      appInfoPanel.setMessage(GctBundle.getString("appengine.infopanel.noproject"),
-          true /* isError*/);
-    }
-  }
+
```

Comment:
nit: Remove extra unnecessary newline here and above, there seems to be 3 here between the constructor and the method below